back/app/models/permission.rb

```diff
@@ -54,6 +54,8 @@ class Permission < ApplicationRecord
   before_validation :update_global_custom_fields, on: :update
 
   def self.available_actions(permission_scope)
+    return [] if permission_scope && !permission_scope.respond_to?(:participation_method)
+
     ACTIONS[permission_scope&.participation_method]
   end
```

Comment:
I tried to come up with a more straightforward version, but was not very successful. Just in case, here's the version I came up with:
```suggestion
  def self.available_actions(permission_scope)
    ACTIONS[permission_scope&.participation_method]
  rescue NoMethodError => e
    e.message == :participation_method ? [] : raise
  end
```